### PR TITLE
[MU4] Fix #305555: add cut2 and cut3 time signatures to master palette

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
resolves https://musescore.org/en/node/305555

This is for master what #7044 was for 3.x

### CODE REVIEW

File: mtest/libmscore/timesig/tst_timesig.cpp [hunk 1]
```diff
@@ -303,11 +309,15 @@ void TestTimesig::timesig_78216()
...
     Measure* m3 = m2->nextMeasure();
+    //Measure* m4 = m3->nextMeasure();
+    //Measure* m5 = m4->nextMeasure();
 
     // verify no timesig exists in segment of final tick of m1, m2, m3
     QVERIFY2(!m1->findSegment(SegmentType::TimeSig, m1->endTick()), "Should be no timesig at the end of measure 1.");
     QVERIFY2(!m2->findSegment(SegmentType::TimeSig, m2->endTick()), "Should be no timesig at the end of measure 2.");
     QVERIFY2(!m3->findSegment(SegmentType::TimeSig, m3->endTick()), "Should be no timesig at the end of measure 3.");
+    //QVERIFY2(!m4->findSegment(SegmentType::TimeSig, m4->endTick()), "Should be no timesig at the end of measure 4.");
+    //QVERIFY2(!m5->findSegment(SegmentType::TimeSig, m5->endTick()), "Should be no timesig at the end of measure 5.");
```
Comment (psmokotnin) [inline mtest/libmscore/timesig/tst_timesig.cpp:320]: Why these lines are commented?

Comment (Jojo-Schmitz) [reply mtest/libmscore/timesig/tst_timesig.cpp:320]: Because no such tests (m4 and m5) have yet been written.

Comment (Jojo-Schmitz) [reply mtest/libmscore/timesig/tst_timesig.cpp:320]: And I just coppied across what I had for MuseScore 3

File: mtest/libmscore/timesig/tst_timesig.cpp [hunk 2]
```diff
@@ -278,10 +278,16 @@ void TestTimesig::timesig10()
     ts2->setSig(Fraction(2, 2), TimeSigType::NORMAL);
     TimeSig* ts3 = new TimeSig(score);
     ts3->setSig(Fraction(4, 4), TimeSigType::FOUR_FOUR);
+    //TimeSig* ts4 = new TimeSig(score);
+    //ts4->setSig(Fraction(2, 2), TimeSigType::CUT_BACH);
```
Comment (Jojo-Schmitz) [inline mtest/libmscore/timesig/tst_timesig.cpp:283]: Missing:
```
    //TimeSig* ts5 = new TimeSig(score);
    //ts4->setSig(Fraction(2, 2), TimeSigType::CUT_TRIPLE);
```

File: mtest/libmscore/timesig/tst_timesig.cpp [hunk 3]
```diff
@@ -303,11 +309,15 @@ void TestTimesig::timesig_78216()
     Measure* m1 = score->firstMeasure();
     Measure* m2 = m1->nextMeasure();
     Measure* m3 = m2->nextMeasure();
+    //Measure* m4 = m3->nextMeasure();
+    //Measure* m5 = m4->nextMeasure();
 
     // verify no timesig exists in segment of final tick of m1, m2, m3
```
Comment (Jojo-Schmitz) [inline mtest/libmscore/timesig/tst_timesig.cpp:315]: `..., m4, m5`

File: mtest/libmscore/timesig/tst_timesig.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [reply mtest/libmscore/timesig/tst_timesig.cpp:320]: Forget what I wrote: these are not needed at all

Comment (Jojo-Schmitz) [reply mtest/libmscore/timesig/tst_timesig.cpp]: removed...

File: mtest/libmscore/timesig/tst_timesig.cpp (hunk 2 above)
Comment (Jojo-Schmitz) [reply mtest/libmscore/timesig/tst_timesig.cpp:285]: added

**Comment by Jojo-Schmitz** [comment]
Funny, the mtests are apparently not run at all (else it would fail here, as it does for #7094 in my first attempt)

File: src/importexport/internal/braille/exportbraille.cpp [hunk 4]
```diff
@@ -859,14 +859,18 @@ QString ExportBraille::brailleTimeSig(TimeSig* timeSig)
...
-        return QString("_c");
-    }
-
     if (timeSig->timeSigType() == TimeSigType::FOUR_FOUR) {
         return QString(".c");
+    } else if (timeSig->timeSigType() == TimeSigType::ALLA_BREVE) {
+        return QString("_c");
     }
-
+#if 0 //ToDo
```
Comment (RomanPudashkin) [inline src/importexport/internal/braille/exportbraille.cpp:867]: Better to remove this code or fix this todo

**Comment by Jojo-Schmitz** [comment]
note to self: rebase needed

**Comment by Jojo-Schmitz** [comment]
See #20196 and #20197, that part needs to get rerverted

**Pull request merged**